Nextest.toml

```diff
@@ -0,0 +1,20 @@
+# This is configuration for `cargo nextest`.
```

Comment:
> probably want a lower case 'N'?

Upper case is better for configuration files, see http://web.mit.edu/rust-lang_v1.25/arch/amd64_ubuntu1404/share/doc/rust/html/cargo/faq.html#why-cargotoml. I want to get to changing `x.toml` to `X.toml` at some point soon.

> also a tad concerned this can't be read from the x.toml. I suspect we want as few config files as possible, and this and x look like they should be able to be merged

Hmm, I actually think the current x.toml is a bit too big and I'd like to split it up at some point. One reason I decided to make nextest's config separate is that I think nextest is going to exist as a project independent of x, and therefore it should have its own config file. (I'd like to make, e.g. a `Hakari.toml` as well with the hakari config.)

> I just went and read some of your lastest changes to diem-devtools and see you recently switched nextest.toml to Nexttest.toml

Yeah I did that because of the link above.

> I think you need to start asking for reviews. There appears to be a lot of unreviewed code in diem-devtools.

I definitely want to do a code deep dive with you (and/or the rest of the team) on nextest very soon, just like I did one on guppy with @bmwill recently.